src/WorldStorage/NBTChunkSerializer.cpp

```diff
@@ -738,6 +738,7 @@ class SerializerCollector:
 			case mtVillager:      EntityClass = "Villager";       break;
 			case mtWitch:         EntityClass = "Witch";          break;
 			case mtWither:        EntityClass = "WitherBoss";     break;
+			case mtWitherSkeleton:        EntityClass = "WitherSkeleton";     break;
```

Comment:
Something's gone wrong with the alignment here.

Reply:
Looks like I forgot to align them entirely.